code/x86-intel/hello-world/hello-world-mac.asm

```diff
@@ -56,7 +56,7 @@ section .text
 _main:
     mov     rax, 0x2000004  ; system call for write. anything with 0x2 is mac specific
     mov     rdi, 1          ; Set output to stdout. 1 = stdout, which is normally connected to the terminal.
-    mov     rsi, msg        ; address of string to output
+    mov     rsi, qword msg  ; address of string to output
```

Comment:
```suggestion
    mov     rsi, qword msg  ; Save the value of msg into rsi. qword is specifying how much data we want to retrieve.
                                           ; Since x86-64 is 64 bit registers, or 8 bytes, we want a qword of data, which is 8 bytes.
```